tests/testutils/iputils.go

```diff
@@ -83,8 +83,32 @@ func GetIPsv6(iface string) ([]string, error) {
 }
 
 func GetGW(iface string, family int) (string, error) {
-	return "", fmt.Errorf("Not implemented")
+	var ret string
+	list, err := net.InterfaceByName(iface)
+	if err != nil {
+		return "", err
+	}
+
+	addrs, err := list.Addrs()
+	if err != nil {
+		return "", err
+	}
+	for _, addr := range addrs {
```

Comment:
I also fail to understand the intention here. The wording of the function doesn't see to match the code.